### DISCUSSION TITLE
Wanting to try on Windows

### DISCUSSION THREAD

**Discussion by delboy1966** [ConSol-Monitoring/snclient#25] [General]
I'd like to test snclient on windows, but I've no idea how to install it as a service like I did with NSClient.
Any help would be appreciated.

**Comment by sni** [comment]
the windows service will be automatically created during installation when using the .msi installer file.

**Reply by delboy1966** [reply]
Yes, I have now found that out, thanks.

I notice its only got the nrpe agent in the ini file, does it not work like NSClient did with the check_nt plugins?

**Reply by sni** [reply]
check_nt is not supported, and probably will never be. You can use check_nrpe or check_nsc_web instead.

**Reply by sni** [reply]
i just checked, and check_nt is even in nsclient marked as deprecated.

**Reply by delboy1966** [reply]
Cool. I'll convert my checks and do some rewriting for the core O/S checks
